cmd/tiller-proxy/internal/handler/apprepos_handler.go

```diff
@@ -256,8 +270,8 @@ func secretForRequest(appRepoRequest appRepositoryRequest, appRepo *v1alpha1.App
 			Name: secretNameForRepo(appRepo.Name),
 			OwnerReferences: []metav1.OwnerReference{
 				metav1.OwnerReference{
-					APIVersion:         appRepo.TypeMeta.APIVersion,
-					Kind:               appRepo.TypeMeta.Kind,
+					APIVersion:         "kubeapps.com/v1alpha1",
+					Kind:               "AppRepository",
```

Comment:
IRL testing showed that `appRepo.TypeMeta` is empty. So given that it's constant, I've updated to the actual values.